ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -867,6 +891,11 @@ export default defineComponent({
       onSave()
     })
 
+    watch(() => getMhrInformation.value.frozenDocumentType,
```

Comment:
Does this need to be watch or can it be a computed value?

Reply:
It is a tricky one, because of `parseMhrInformation` that's happening in the `onMounted`, we need to get the `frozenDocumentType` at a later time, hence the watch.